sigstore-java/src/main/java/dev/sigstore/rekor/client/RekorResponse.java

```diff
@@ -56,7 +57,12 @@ public interface RekorResponse {
   static RekorResponse newRekorResponse(URI entryLocation, String rawResponse)
       throws RekorParseException {
     var type = new TypeToken<Map<String, RekorEntry>>() {}.getType();
-    Map<String, RekorEntry> entryMap = GSON.get().fromJson(rawResponse, type);
+    Map<String, RekorEntry> entryMap;
+    try {
+      entryMap = GSON.get().fromJson(rawResponse, type);
+    } catch (JsonSyntaxException jse) {
+      throw new RekorParseException("Rekor entry json could not be parsed", jse);
```

Comment:
WDYT of mentioning the problematic response in the error message?

Reply:
I think the stack trace should show it? but yeah I can just do jse.getMessage() too. I'll add that in.